src/scss/components/infinite_table.scss

```diff
@@ -8,12 +8,11 @@
 	&.ui.primary.basic {
 	    box-shadow: 0px 0px 0px 0px white inset !important;
 	    font-weight: bold !important;
-		text-align: center;
+		text-align: center !important;
 		color: #1E5AA8 !important;
 	}
+}
 
-	&.no-show {
-		background-color: #c2c2c2 !important;
-		pointer-events: none;
-	}
+.minus.icon {

```

Comment:
Make this less generic. This selector should affect only the cells in the rankings table.